packages/react/src/List/OrderedList.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import {ListContext} from './UnorderedList'
+
+import styles from './List.module.css'
+
+export type OrderedListProps = BaseProps<HTMLElement> & {
+  /**
+   * React.ReactNode and React.ReactNode[] are valid children.
+   */
+  children?: React.ReactNode | React.ReactNode[]
+  /**
+   * Ref object to be attached to the list.
+   */
+  ref?: Ref<HTMLOListElement>
```

Comment:
Related comment https://github.com/primer/brand/pull/145/files#r1053562728

Reply:
In next push